auto-self-nominate/package.json

```diff
@@ -9,7 +9,12 @@
     "fmt": "tslint -p . --fix && prettier 'src/**/*.ts' --write"
   },
   "dependencies": {
-    "codechain-sdk": "^1.2.0"
+    "@sendgrid/mail": "^6.4.0",
+    "@slack/client": "^5.0.1",
+    "@types/lodash": "^4.14.134",
+    "D": "^1.0.0",
```

Comment:
"This package is no longer supported and has been deprecated." https://www.npmjs.com/package/D
Why do you use it?

Reply:
Sorry, I typed `yarn add D` by mistake while typing command to add devDependencies: starting with `yarn add -D`.